### PR TITLE
Updated part categories for 1.2

### PR DESCRIPTION
**Pull request by IronCretin**
Updated part configs for the new 1.2 categories.

### CODE REVIEW

File: Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg [hunk 1]
```diff
@@ -16,7 +16,7 @@ MODEL
 	TechRequired = specializedConstruction
 	entryCost = 8000
 	cost = 2530
-	category = Structural
+	category = Development -- I think you have a good idea with the first paragraph, and it definetly has a place, but its presence at the start is a bit jarring. Perhaps rewrite it into the first or move the thesis up. Either way, the first and second paragraphs together make a good opening with a great hook
```
Comment (jsolson) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg]: I think your clipboard ran a muck on this one.

Comment (IronCretin) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg]: Oh, crap, that's embarrassing. That'll teach me to try to do two things at once.

Comment (IronCretin) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg]: How do I fix this in the pull?

Comment (CobaltWolf) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4_Interstage.cfg]: yeah, what? also this review system is nice

**Comment by CobaltWolf** [comment]
I'm not sure about the resupply containers. I think they should stay in Utility, at least until we see what KIS does.

**Comment by IronCretin** [comment]
Don't they carry payload? The category is for cargo bays, fairings, and service bays.

**Comment by CobaltWolf** [comment]
They're KIS containers. The unpressurized module would belong there, I want to see what the KIS devs do for the rest of them.

**Pull request merged**